content/getting-started/connect-the-pim-4x/welcome.md

```diff
@@ -0,0 +1,33 @@
+# 👋 Welcome in this tutorial!
+
+This tutorial will teach you:
+> How to connect the PIM with the API on a 4.x PIM
+
+## Goal
+At the end of this tutorial, you will know what are the primary steps to properly connect the PIM.
+
+::: tips
+Connections will be achieved with the help of the API, as the API is the best way to connect the PIM. Wondering why? Discover the reasons [here](/documentation/introduction.html#why-should-you-use-our-api)!
+:::
+
+## Prerequisites
+
+<i class="fa fa-check-square"></i> **You have access to a 4.x Akeneo PIM.**  
+This can seem pretty obvious. But yeah! You will need a PIM if you want to connect it to any third parties.  
+And of course, you also have to make sure that you can access it from where you are requesting.
```

Comment:
I would add something like:
If you do not have access to a 4.x version of Akeneo PIM, please get in touch with Akeneo team, or your Channel Manager, etc.

Reply:
You're thinking about which kind of persona here?

Reply:
As a System Integrator or a Technology Provider, I want to develop a connector but do not have access to the latest Akeneo version.
-> OK then ask your Channel Manager (I'm considered as one in the Portal) or get in touch with partners@akeneo.com.
Thanks :)

Reply:
```suggestion
And of course, you also have to make sure that you can access it from where you are requesting.

::: tips
If you do not have access to a 4.x version of Akeneo PIM, please get in touch with [Akeneo team](mailto:partners@akeneo.com) or your Channel Manager. 
:::
```